hastexo_guacamole_client/settings.py

```diff
@@ -36,12 +31,28 @@ def get_env_setting(setting):
     },
     'root': {
         'handlers': ['console'],
-        'level': 'DEBUG' if DEBUG else os.getenv('DJANGO_LOG_LEVEL',
-                                                 'WARNING').upper(),
+        'level': 'DEBUG' if DEBUG else DJANGO_LOG_LEVEL.upper()
     },
 }
 
-ALLOWED_HOSTS = os.getenv('HASTEXO_GUACAMOLE_ALLOWED_HOSTS', "")
+ALLOWED_HOSTS = os.environ.get('HASTEXO_GUACAMOLE_ALLOWED_HOSTS', '')
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': os.environ.get('HASTEXO_GUACAMOLE_DEFAULT_DB_NAME', 'edxapp'),
+        'USER': os.environ.get('HASTEXO_GUACAMOLE_DATABASE_USER', 'edxapp001'),
+        'PASSWORD': os.environ.get(
+            'HASTEXO_GUACAMOLE_DATABASE_PASSWORD', 'password'),
+        'HOST': os.environ.get('HASTEXO_GUACAMOLE_DATABASE_HOST', 'localhost'),
+        'PORT': int(os.environ.get('HASTEXO_GUACAMOLE_DATABASE_PORT', '3306')),
+        'ATOMIC_REQUESTS': os.environ.get(
```

Comment:
Here too I think you need `bool(strtobool())`, otherwise the string `false` will be considered a truthy value.

Reply:
Ack, done.